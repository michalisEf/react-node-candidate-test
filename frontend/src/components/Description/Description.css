.description {
  line-height: 1.3rem;
  font-size: .9rem;
}
