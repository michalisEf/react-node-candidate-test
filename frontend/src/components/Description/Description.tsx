import React from 'react';

import { IDescription } from '../../interfaces';

import './Description.css';

const Description = ({ description }: IDescription): JSX.Element => {
  return (
    <p className="description">{description}</p>
  );
};

export default Description;
