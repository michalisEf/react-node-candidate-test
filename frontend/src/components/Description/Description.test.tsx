import React from 'react';
import { render, screen } from '@testing-library/react'

import Description from './Description';

it('renders Description component', () => {
	const props = {
		description: "This is generic description",
	};
	render(<Description { ...props }/>);
	const desc = screen.getByText(/This is generic description/);
	expect(desc).toBeInTheDocument();
	expect(desc).toMatchSnapshot();
});