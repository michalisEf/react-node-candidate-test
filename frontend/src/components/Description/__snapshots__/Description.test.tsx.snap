// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Description component 1`] = `
<p
  class="description"
>
  This is generic description
</p>
`;
